test_util/launch_configs/README.md

```diff
@@ -0,0 +1,50 @@
+# DC/OS Launch Configuration YAML
+## Design Intention
+The intention of this configuration file is to provide an interface by which
+all deployments of DC/OS, regardless of provider have a similar format, thus
+complementing the goal of dcos-launch to provide a single tool for launching
+across a variety of provider APIs
+
+## Supported Deployments and Examples
+- [Simple AWS Cloudformation](aws-cf.yaml)
+- [Zen AWS Cloudformation](aws-zen-cf.yaml)
+- [Non-DC/OS AWS Linux Cluster](aws-linux-cluster.yaml)
+- [Onprem Install on AWS Linux Cluster](aws-onprem.yaml)
+- [Onprem Install on Previously Provisioned Linux Cluster](onprem-from-info.yaml)
+- [Azure Template Deployment](azure.yaml)
+
+## Keywords and Definitions
+
+### Required for Deployment
+* `deployment_type`: Can be one of `template`, `onprem`, or `linux-cluster`.
+  * `template` refers to either an AWS cloudformation template or an Azure Resource Manager deployment template. Use of this options will require setting `deployment_name`, `template_url`, `provider`, and `provider_info`
+  * `linux_cluster` will used hard-coded templates to deploy a cluster of linux machines in a shared network on the provider of choice (currently only supports AWS). `deployment_name` and `provider` must be set. See `linux_cluster` below for options. Only `aws` provider is currently supported for this deploy option
```

Comment:
I like `linux` as its a bit cleaner, but when you fully write it out, you get:
{code}
---
deployment_type: onprem
provider: aws
linux:
   instance_type: t2.micro
   os_name: cent-os-7
{code}

So then it gets awkward that you have linux-> instance_type because instances don't really exist in the linux namespace.

IMO the real issue here is I just can't think of a good word to describe "a group of machines, all of which have the same OS installed and are provided by some cloud service". `cloud_cluster` ? And then maybe something like:
{code}
---
deployment_type: cloud_cluster
provider: aws
cloud_cluster:
  linux: cent-os-7
  host_count: 4
```
which would be nice because then we have the freedom to put ```ami:``` under there or even ```windows: ```

What do you think?